opnode/rollup/derive/payload_attributes.go

```diff
@@ -77,12 +77,12 @@ func (dep *L1InfoDepositSource) SourceHash() common.Hash {
 //
 // Additionally, the event log-index and
 func UnmarshalLogEvent(ev *types.Log) (*types.DepositTx, error) {
-	if len(ev.Topics) != 3 {
-		return nil, fmt.Errorf("expected 3 event topics (event identity, indexed from, indexed to)")
-	}
 	if ev.Topics[0] != DepositEventABIHash {
 		return nil, fmt.Errorf("invalid deposit event selector: %s, expected %s", ev.Topics[0], DepositEventABIHash)
 	}
+	if len(ev.Topics) != 3 {
```

Comment:
Please revert the order of the checks. The length of topics may be 0 (see LOG0 opcode)